internal/core/items/path.rs

```diff
@@ -120,13 +120,14 @@ impl Item for Path {
         self: Pin<&Self>,
         backend: &mut ItemRendererRef,
         self_rc: &ItemRc,
+        size: LogicalSize,
     ) -> RenderingResult {
         let clip = self.clip();
         if clip {
             (*backend).save_state();
-            (*backend).combine_clip(self.geometry(), LogicalLength::zero(), LogicalLength::zero());
+            (*backend).combine_clip(size.into(), LogicalLength::zero(), LogicalLength::zero());
```

Comment:
This is a behavior change, but the new behavior is correct.
The clip is already in the item coordinate so it shouldn't be offset by x,y

Reply:
Good catch. Changelog worthy?